generate.sh

```diff
@@ -28,8 +28,8 @@ function generateDocs {
     git pull origin $1
     sleep 3
     git submodule update --init --recursive --force
-    php ../app/composer.phar self-update
-    php ../app/composer.phar install
+    composer update
```

Comment:
This is actually a major change. Before composer was only updating itself. Now it will update all used packages. Not sure if that was changed on purpose.

Reply:
Yes, that purpose. I think it's basic to install the PHP Composer and it's mostly installed on a PHP developers system. I was quite confused to have a separate step to install it. Also the checksum does not match necessarily. Also you can't say if the newest version works every time. In the README I wrote that one should install PHP Composer.

Reply:
Ok. But if we expect composer to be installed, shouldn't a `composer install` be enough? I wonder why generating the docs should execute a `composer update` to update all dependencies of Matomo. This could actually break things if a newer version of a dependency might cause problems.

Reply:
The composer update was necessary on my machine, because composer.lock was out of sync

Reply:
Removed the composer update. Anything else?